docs/sources/enterprise/reporting.md

```diff
@@ -39,10 +39,45 @@ Currently only Organization Admins can create reports.
 1. Select the layout option for generated report: **Portrait** or **Landscape**.  
 1. Enter scheduling information. Options vary depending on the frequency you select.
 1. **Save** the report.
+1. **Send test mail** after saving the report to verify that the report looks like you expect it to.
```

Comment:
I would like to add that it also can be used to verify that emails actually can be sent. Not sure how to phrase it though

Reply:
```suggestion
1. **Send test mail** after saving the report to verify that the report works like you expect it to.
```

mayhaps?

Reply:
```suggestion
1. **Send test mail** after saving the report to verify that the whole configuration is working as expected.
```
Maybe? I agree that otherwise it's a bit redundant with the previous step `Preview the report to make sure it appears as you expect. Update if necessary`
And I think it should be "send test email" but this is what in the UI. Should we update it?